salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -80,12 +80,13 @@ server {
 
     ## for accessing it with Selenium
     {% if pillar.elife.env == 'dev' %}
-    location /app_test.php {
+    location /app.php {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         include fastcgi_params;
-        fastcgi_param SCRIPT_FILENAME /srv/journal/web/app_test.php;
+        fastcgi_param SCRIPT_FILENAME /srv/journal/web/app.php;
         fastcgi_param DOCUMENT_ROOT $realpath_root;
         fastcgi_param ENVIRONMENT_NAME {{ pillar.elife.env }};
+        fastcgi_param APP_ENV {{ pillar.elife.env }};
```

Comment:
if this is specific to Selenium, should it be hardcoded to `test`?

Reply:
Haven't tried but I think this is broken: needs to be two separate sites. Not sure it's worth fixing, only used inside the VM which is unused and to shortly be replaced?

Reply:
Though `location /app_test.php` could remain the `APP_ENV` set to `dev`? `behat.yml` needs reverting too in that case.

Reply:
Given we are not running any test in the Vagrant VM and this doesn't influence `journal--ci` (it's a `dev` block), this section can be removed altogether if it makes things easier.